test/correctness/simd_op_check.cpp

```diff
@@ -1717,31 +1717,31 @@ struct Test {
         check("vmin(v*.w,v*.w)", hvx_width/4, min(i32_1, i32_2));
 
         check("vcmp.gt(v*.b,v*.b)", hvx_width/1, select(i8_1 < i8_2, i8_1, i8_2));
-        check("vcmp.gt(v*.ub,v*.ub)", hvx_width/1, select(u8_1 < u8_2, u8_1, u8_2));
-        check("vcmp.gt(v*.h,v*.h)", hvx_width/2, select(i16_1 < i16_2, i16_1, i16_2));
-        check("vcmp.gt(v*.uh,v*.uh)", hvx_width/2, select(u16_1 < u16_2, u16_1, u16_2));
-        check("vcmp.gt(v*.w,v*.w)", hvx_width/4, select(i32_1 < i32_2, i32_1, i32_2));
+        check("vcmp.gt(v*.ub,v*.ub)", hvx_width/1, select(u8_1 < u8_2, u8_3, u8_4));
```

Comment:
Maybe leave the false case of this one to be u8_2 for consistency with the rest? Inconsistencies like this can sometimes be tricky when revisiting this code.

Reply:
Yes, will do. I had changed all 8 bit tests to _3 and _4 and then realized _3 was enough. So, I changed them to remove _4.  Looks like i missed this one.